src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -1548,6 +1557,26 @@ subroutine CanopyFluxes(bounds,  num_exposedvegp, filter_exposedvegp,
          call PhotosynthesisTotal(fn, filterp, &
               atm2lnd_inst, canopystate_inst, photosyns_inst)
          
+         ! Calculate water use efficiency
+         !   does not support multi-layer canopy
+         if (nlevcan == 1) then 
+            do f = 1, fn
+               p = filterp(f)
+               c = patch%column(p)
+               g = patch%gridcell(p)
+               
+               if ( is_near_local_noon( grc%londeg(g), deltasec=3600 ) .and. fpsn(p)>0._r8 )then
+                  gs        = 1.e-6_r8*(laisun(p)*gs_mol_sun(p,iv)+laisha(p)*gs_mol_sha(p,iv))    ! 1e-6  converts umolH2O->molH2O
+                  if ( gs>0._r8 ) then
+                     iwue_ln(p)   = fpsn(p)/gs
+                  else
+                     iwue_ln(p)   = spval
+                  end if
+               else
+                  iwue_ln(p)   = spval
+               end if
+            end do
```

Comment:
We should add an "else" statement with an endrun so that it aborts if levcan is ever set to 1. We won't remember this detail when that happens, so need a reminder. Since, the endrun would be outside the loop that should be fine for efficiency.

- [ ] Add endrun if levcan /= 1